core/scheduler/service/replay_service.go

```diff
@@ -68,7 +68,12 @@ func (r ReplayService) GetReplayList(ctx context.Context, projectName tenant.Pro
 }
 
 func (r ReplayService) GetReplayByID(ctx context.Context, replayID uuid.UUID) (*scheduler.ReplayWithRun, error) {
-	return r.replayRepo.GetReplayByID(ctx, replayID)
+	replayWithRun, err := r.replayRepo.GetReplayByID(ctx, replayID)
+	if err != nil {
+		return nil, err
+	}
+	replayWithRun.Runs = scheduler.JobRunStatusList(replayWithRun.Runs).GetSortedRunsByScheduledAt()
```

Comment:
do we need this changes for moving namespace for a resource?

Reply:
no , i think this has come from base branch , will fix this